recipe/meta.yaml

```diff
@@ -33,11 +21,10 @@ requirements:
     - python
     - pip
     - cython
-    - proj4
+    - proj4 6.0.*
   run:
     - python
-    - numpy
-    - proj4
+    - proj4 >=6.0
```

Comment:
Sorry, I forgot but this should have an upper version limit. Looking at https://abi-laboratory.pro/?view=timeline&l=proj I believe that the best we can do to pin to the exact version, so both run and build would be `proj4 6.0.0`.

@isuruf do you agree?

Reply:
Yes. `{{ pin_compatible("proj4", max_pin="x.x.x") }}`

Reply:
I added the pin 6.0.0. Is it correct as is?

Reply:
Good enough for now but we should do `{{ pin_compatible("proj4", max_pin="x.x.x") }}` for `run` and modify the global pinning. I'll take care of that later.

Let's get this merged and see how many packages that depends on `pyproj` we are going to break :stuck_out_tongue_winking_eye: